nemo/collections/tts/models/hifigan.py

```diff
@@ -58,32 +57,41 @@ def __init__(self, cfg: DictConfig, trainer: 'Trainer' = None):
         self.discriminator_loss = DiscriminatorLoss()
         self.generator_loss = GeneratorLoss()
 
+        self.l1_factor = 45
+        self.num_log_audios = 5
```

Comment:
You should add this in the yaml instead of hard coding these.

Reply:
Ok.